website/src/components/Home/ProjectSection.js

```diff
@@ -5,23 +5,16 @@ import ProjectCard from "../common/ProjectCard";
 export default function Project() {
   return (
     <div className='project-section'>
-      <h1 className='title'> Our Projects</h1>
-      <p className='main-text'>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-        veniam, quis nostrud exercitation ullamco laboris nisi ut.
-      </p>
-      <p className='extra-text'>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-        veniam, quis nostrud exercitation ullamco laboris nisi ut.
-      </p>
-      <p>Check out our current projects!</p>
-      <div className='projects'>
-        <ProjectCard />
-        <ProjectCard />
-        <ProjectCard />
-      </div>
+    <h1 className='title'> Our Projects</h1>
+    <p className='main-text'>
+    Each quarter, our team works closely with a non-profit organization to develop software that will help them better serve their communities. 
+    </p>
+    <a className="project-link" href="/projects">Check out our current projects!</a>
```

Comment:
I would say "most recent" instead of "current" projects since we don't have a current one right now and we also won't have multiple ongoing ones in a while. Hyperlink to projects page probs not necessary either